src/Public/OpenApi.ps1

```diff
@@ -1516,39 +2307,206 @@ function Enable-PodeOpenApiViewer
+            OpenApi  = $meta.OpenApi
             DarkMode = $meta.DarkMode
         }
     }
+}
+
+
+
+<#
+.SYNOPSIS
+Adds an external docs reference. 
+
+.DESCRIPTION
+Creates a new  reference from another OpenAPI schema.
+
+.PARAMETER Name
+The Name of the reference.
+
+.PARAMETER url
+The link to the external documentation
+
+.PARAMETER Description
+A Description of the external documentation.
+
+.PARAMETER Array
+If supplied, the schema reference will be treated as an array.
+
+.EXAMPLE
+Add-PodeOAExternalDoc  -Name 'SwaggerDocs' -Description 'Find out more about Swagger' -Url 'http://swagger.io'
+#>
+function Add-PodeOAExternalDoc
+{ 
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Name,
+
+        [Parameter(Mandatory = $true)]
+        [ValidateScript({ $_ -match '^https?://.+' })]
+        $Url,
+
+        [Parameter()]
+        [string]
+        $Description
+    )
+
+    $param = @{}
+
+    if ($Description)
+    {
+        $param.description = $Description
+    }
+    $param['url'] = $Url
+    $PodeContext.Server.OpenAPI.hiddenComponents.externalDocs[$Name] = $param
+    
+}
+
+
+<#
+.SYNOPSIS
+Creates a OpenAPI Tag reference property.
+
+.DESCRIPTION
+Creates a new OpenAPI tag reference.
+
+.PARAMETER Name
+The Name of the tag. 
+
+.PARAMETER Description
+A Description of the tag.
+
+.PARAMETER ExternalDocs
+If supplied, the tag reference to an existing external documentation reference.
+
+.EXAMPLE
+Add-PodeOATag -Name 'store' -Description 'Access to Petstore orders' -ExternalDocs 'SwaggerDocs'
+#>
+function Add-PodeOATag
+{ 
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Name, 
+
+        [Parameter()]
+        [string]
+        $Description,
+
+        [Parameter()]
+        [string]
+        $ExternalDocs
+    )
+    
+    $param = @{ 
+        'name' = $Name 
+    }
+
+    if ($Description)
+    {
+        $param.description = $Description
+    }
+
+    if ($ExternalDocs)
+    {
+        if ( !(Test-PodeOAExternalDoc -Name $ExternalDocs))
+        {
+            throw "The ExternalDocs doesn't exist: $ExternalDocs"
+        }  
+        $param.externalDocs = $PodeContext.Server.OpenAPI.hiddenComponents.externalDocs[$ExternalDocs]
+    }
+
+    $PodeContext.Server.OpenAPI.tags[$Name] = $param
+
+}
+
+
+
+function New-PodeOAExtraInfo
+{
+    param(
+        [Parameter()]
+        [ValidateScript({ $_ -match '^https?://.+' })]
+        [string]$TermsOfService,
+
+        [Parameter(Mandatory = $true)] 
+        [string]$License,
+
+        [Parameter(Mandatory = $true)]
+        [ValidateScript({ $_ -match '^https?://.+' })]
+        [string]$LicenseUrl,
+
+        [Parameter()]
+        [string]$ContactName,
+
+        [Parameter()]
+        [ValidateScript({ $_ -match '^\w+([-+.]\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*$' })]
+        [string]$ContactEmail,
+
+        [Parameter()]
+        [ValidateScript({ $_ -match '^https?://.+' })]
+        [string]$ContactUrl  
```

Comment:
Drop the parameter names onto the next lines 😄

Reply:
done